### ISSUE TITLE
Extension URL's are appended to URL params

### ISSUE THREAD

**Issue by lenvanessen** [bolt/bolt#7113]
Okay, to reproduce:
1. Install a clean Bolt 3.4
2. Install Bolt Translate (Bolt Translate adds a URL-param `_locale` with Symfony's UrlGeneratorInterface
3. Install any other extension, like Labels or Menu-editor
4. The extension URL get's appended to the `_locale` parameters (and as far as I can tell any URL-parameters.
5. Url's come out like `bolt/extensions?_locale=en/menueditor`

The expected outcome would be `bolt/extensions/menueditor?_locale=en`

**Comment by lenvanessen** [comment]
Test case incoming...

**Comment by GawainLynch** [comment]
This behaves as expected, and the element is created as `<a href="/bolt/extensions/menueditor?koala=gumleaves">`

```php
<?php

namespace Koala;

use Bolt\Extension\SimpleExtension;
use Bolt\Menu\MenuEntry;

class DropBear extends SimpleExtension
{
    protected function registerMenuEntries()
    {
        return [
            MenuEntry::create('test')->setLabel('Test')->setRoute('_bolt_extensions_menueditor', ['koala' => 'gumleaves'])
        ];
    }
}

```

**Comment by lenvanessen** [comment]
Hi Gawain, 

Yes, if you register the param with the MenuEntry class then it works here as well.
But, if they are registered globally extending the url_generator, then it breaks.

I made a test case:
https://github.com/lenvanessen/TestCasE

If it's total koala logic, apologies in advance.

**Comment by GawainLynch** [comment]
OK, so to the above test case that I included, I added your `UrlOverrider` class and copied your `registerServices()` method, and I get this element:

`<a href="/bolt/extensions/menueditor?koala=gumleaves&amp;_bug=1">`

**Comment by GawainLynch** [comment]
In the Label extension, adding ` ->setRoute('labels')` works:
```php
    protected function registerMenuEntries()
    {
        $app = $this->getContainer();

        if (!$app['labels.config']->isShowMenu()) {
            return [];
        }

        return [
            (new MenuEntry('labels', 'labels'))
                ->setRoute('labels')
                ->setLabel('Label translations')
                ->setIcon('fa:flag')
                ->setPermission('labels'),
        ];
    }
```

**Comment by GawainLynch** [comment]
Fixed in #7115
